schemas/preferences.sublime-settings.json

```diff
@@ -5,964 +5,975 @@
-    "match_brackets_content": {
-      "markdownDescription": "Set to `false` if you'd rather only highlight the brackets when the caret is next to one.",
-      "type": "boolean",
-      "default": true
-    },
-    "match_brackets_square": {
-      "markdownDescription": "Set to `false` to not highlight square brackets. This only takes effect if `match_brackets` is `true`.",
-      "type": "boolean",
-      "default": true
-    },
-    "match_brackets_braces": {
-      "markdownDescription": "Set to `false` to not highlight curly brackets. This only takes effect if `match_brackets` is `true`.",
-      "type": "boolean",
-      "default": true
-    },
-    "match_brackets_angle": {
-      "markdownDescription": "Set to `false` to not highlight angle brackets. This only takes effect if `match_brackets` is `true`.",
-      "type": "boolean",
-      "default": true
-    },
-    "match_tags": {
-      "markdownDescription": "Enables visualization of the matching tag in HTML and XML.",
-      "type": "boolean",
-      "default": true
-    },
-    "match_selection": {
-      "markdownDescription": "Highlights other occurrences of the currently selected text.",
-      "type": "boolean",
-      "default": true
-    },
-    "line_padding_top": {
-      "markdownDescription": "Additional spacing at the top of each line, in pixels.",
-      "type": "integer",
-      "default": 0
-    },
-    "line_padding_bottom": {
-      "markdownDescription": "Additional spacing at the bottom of each line, in pixels.",
-      "type": "integer",
-      "default": 0
-    },
-    "scroll_past_end": {
-      "markdownDescription": "Set to `false` to disable scrolling past the end of the buffer.\nOn Mac, this value is overridden in the platform specific settings, so you'll need to place this line in your user settings to override it.",
-      "type": "boolean",
-      "default": true
-    },
-    "scroll_context_lines": {
-      "markdownDescription": "Set the number of context lines to show when scrolling. This affects all selection changes, like selection dragging, page-up/page-down and moving the caret.",
-      "type": "integer",
-      "default": 0
-    },
-    "move_to_limit_on_up_down": {
-      "markdownDescription": "This controls what happens when pressing up or down when on the first or last line.\nOn Mac, this value is overridden in the platform specific settings, so you'll need to place this line in your user settings to override it.",
-      "type": "boolean",
-      "default": false
-    },
-    "mouse_wheel_switches_tabs": {
-      "markdownDescription": "Linux only.",
-      "type": "boolean",
-      "default": true
-    },
-    "draw_white_space": {
-      "markdownDescription": "Controls when white space is drawn. Any of the valid options may be combined.\nFor example [\"selection\", \"trailing\", \"isolated\"] will draw whitespace at the end of any lines and under the selection.",
-      "oneOf": [
-        {
+  "definitions": {
+    "editor_settings": {
```

Comment:
For definitions, I think it's common to use CamelCase convention like `EditorSettings`.